client/ayon_core/plugins/publish/integrate_review.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import pyblish.api
+import ayon_api
+from ayon_api.server_api import RequestTypes
+
+from ayon_core.lib import get_media_mime_type
+from ayon_core.pipeline.publish import get_publish_repre_path
+
+
+class IntegrateAYONReview(pyblish.api.InstancePlugin):
+    label = "Integrate AYON Review"
+    # Must happen after IntegrateNew
+    order = pyblish.api.IntegratorOrder + 0.15
+
+    def process(self, instance):
+        project_name = instance.context.data["projectName"]
+        src_version_entity = instance.data.get("versionEntity")
+        src_hero_version_entity = instance.data.get("heroVersionEntity")
+        for version_entity in (
+            src_version_entity,
+            src_hero_version_entity,
+        ):
+            if not version_entity:
+                continue
+
+            version_id = version_entity["id"]
+            self._upload_reviewable(project_name, version_id, instance)
+
+    def _upload_reviewable(self, project_name, version_id, instance):
+        ayon_con = ayon_api.get_server_api_connection()
+        major, minor, _, _, _ = ayon_con.get_server_version_tuple()
+        if (major, minor) < (1, 3):
+            self.log.info(
+                "Skipping reviewable upload, supported from server 1.3.x."
+                f" User server version {ayon_con.get_server_version()}"
+            )
+            return
+
+        uploaded_labels = set()
+        for repre in instance.data["representations"]:
+            repre_tags = repre.get("tags") or []
+            # Ignore representations that are not reviewable
+            if "webreview" not in repre_tags:
+                continue
+
+            # exclude representations with are going to be published on farm
+            if "publish_on_farm" in repre_tags:
+                continue
```

Comment:
In some cases the targeting to the farm is stored on `instance.data["farm"] = True` (no idea whether those have any representation data to begin with, but I've never seen the "publish_on_farm" tag on representations so I'm unsure whether there may be cases where `instance.data["farm"] = True` but the `representations` data may still be defined?

Reply:
I don't know. I've copied ftrack review logic and changed `ftrackreview` to `webreview`.